src/viper/viper/api/transform/ColToSilver.scala

```diff
@@ -39,6 +39,8 @@ case class ColToSilver(program: col.Program[_]) {
   val currentMapGet: ScopedStack[col.MapGet[_]] = ScopedStack()
   val currentDividingExpr: ScopedStack[col.DividingExpr[_]] = ScopedStack()
 
+  var inTriggers = false
```

Comment:
please use a `ScopedStack` for this, I plan to enable parallelism within rewriters at some point.